src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -274,6 +274,31 @@ public function monthlyOn($day = 1, $time = '0:0')
     }
 
     /**
+     * Schedule the event to run on the last day of the month at a given time.
+     *
+     * @param  string  $time
+     * @return $this
+     */
+    public function endOfMonth($time = '0:0')
```

Comment:
If you want to month a monthly report it seems to me you would just run `monthly()` on the scheduler, which will run at midnight of first day of month.